plenum/cli/cli.py

```diff
@@ -78,7 +79,7 @@
 from plenum.common.log import CliHandler, getlogger, Logger, \
     getRAETLogLevelFromConfig, getRAETLogFilePath, TRACE_LOG_LEVEL
 from plenum.server.node import Node
-from plenum.common.types import CLIENT_STACK_SUFFIX, NodeDetail, HA
+from plenum.common.types import NodeDetail, HA
```

Comment:
Agree. But I think such refactoring is not in the scope of this story... I will create a task in backlog to got through types/constants and rename them to be more clear